packages/upstream/ppx_deriving_rpc.5.8.0/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/ocaml-rpc/archive/v5.8.0.tar.gz"
+checksum: "3ba9648e65197641a38ccfedb9135b70"
```

Comment:
The checksum is wrong here.
Can't we use some automated tool to update the version here? (maybe that daily script that we used to run that checks for new upstream versions?) Looks like it is very error prone to do this manually.

Reply:
We are not yet publishing ocaml-rpc upstream. The version there is way outdated. After my idl changes are merged I'll release it upstream and will try to keep the release synced.

Reply:
It is probably possible to use dune-release (or a slightly patched version) to do it though

Reply:
I was thinking of updating our python script in xs-opam so that it can look at a source repo and update it to the latest release in the source repo of the package